exercises/roman-numerals/src/lib.rs

```diff
@@ -1 +1,15 @@
+use std::fmt::{Display, Formatter, Result};
 
+pub struct Roman;
+
+impl Display for Roman {
+    fn fmt(&self, _f: &mut Formatter) -> Result {
+        unimplemented!("Return a roman-numeral string representation of the Roman object");
+    }
+}
+
+impl From<i32> for Roman {
```

Comment:
Given that I've never heard of a negative roman numeral, and none of the tests use negative numbers, it's probably better to implement `From<u32>`.